IdentityCore/src/requests/broker/MSIDSSOExtensionInteractiveTokenRequest.m

```diff
@@ -196,7 +196,7 @@ - (ASPresentationAnchor)presentationAnchor
         return anchor;
     }
     
-    return self.requestParameters.parentViewController.view.window;
+    return self.requestParameters.parentViewController ? self.requestParameters.parentViewController.view.window : self.requestParameters.presentationAnchorWindow;
```

Comment:
Should we pass same window for system browser calls?

Reply:
OneAuth is not using system browser at this moment.

My worry is if there are partners using this API without providing anchor, our current logic will automatically fall to use keywindow. 

With new logic, we still reply on partners to pass in a fall back window. Worst case is the system browser will present at random places without further integration.